test/Integration/IntegrationTestFixture.cs

```diff
@@ -66,7 +66,10 @@ public IntegrationTestFixture()
         private void StartFunctionHosts()
         {
             string binPath = TestUtils.GetPathToBin();
-            foreach (SupportedLanguages lang in Enum.GetValues(typeof(SupportedLanguages)))
+            string languages = Environment.GetEnvironmentVariable("LANGUAGES_TO_TEST");
```

Comment:
Could you add a comment here explaining why we're doing this? Will help make sure we don't break it in the future 😄